### ISSUE TITLE
InnerBlocks edit as HTML option creates invalid content

### ISSUE THREAD

**Issue by virgiliud** [WordPress/gutenberg#11438]
When selecting an innerBlock container, like a single column from the column block, we have the option to "edit as HTML". If you click out of the inner block while editing as HTML it will produce an "unexpected or invalid content" error.

Is the edit as HTML option useful for inner block containers? It seems like it breaks the code. 

I'm using Gutenberg version 4.1.1. and Chrome 70.0.3.

Screenshots testing on Columns block:

![screen shot 2018-11-02 at 10 28 09 pm](https://user-images.githubusercontent.com/3076742/47939414-e8dabf00-deef-11e8-959f-8d19324fe020.png)
![screen shot 2018-11-02 at 10 27 39 pm](https://user-images.githubusercontent.com/3076742/47939415-e8dabf00-deef-11e8-8634-6936ddbe8efb.png)

**Comment by designsimply** [comment]
Tested and confirmed that clicking on the white space of the editor page while in "Edit as HTML" mode for a Columns block results in the block becoming invalid with this error: "This block contains unexpected or invalid content." ([42s](https://cloudup.com/cdDSvuvTb8c))

<sup>Tested using WordPress 4.9.8 and Gutenberg 4.2.0-rc.1 with Firefox 63.0 on macOS 10.13.6.</sup>

**Comment by randomtu** [comment]
I can reproduce the issue on any blocks with the InnerBlocks component. It will work properly again when you click the undo button.after getting the error. 

Had to disable the textarea to at least mitigate the issue. I don't really want to edit the html anyway.

scss:
```
div[data-type="block-namespace/block-name"] {
    textarea.editor-block-list__block-html-textarea {
	pointer-events: none;
    }
}
```

**Comment by websupporter** [comment]
I can confirm this issue. Happens also with the general columns block (the wrapper for the single columns)

**Comment by ZebulanStanphill** [comment]
This should be moved out of the "WordPress 5.0.x Follow Ups" milestone since it was resolved before 5.0 thanks to #11785.

**Comment by miesismore** [comment]
Same problem here, really frustrating, seriously we can't Justify a text??
